data/docs/logs-management/send-logs/opentelemetry-nodejs-bunyan-logs.mdx

```diff
@@ -0,0 +1,286 @@
+  res.status(400).send("Warn log endpoint");
+});
+
+app.listen(3000, () => {
+  logger.info("Server listening on port 3000");
+});
+```
+
+### Step 4: Configure SigNoz Endpoint
+
+Update the `instrumentation.js` file with your SigNoz endpoint:
+
+<Tabs>
+<TabItem value="vm" label="VM / Local" default>
+
+```javascript:instrumentation.js
+const logExporter = new OTLPLogExporter({
+  url: 'https://ingest.<region>.signoz.cloud:443/v1/logs', // OTLP logs endpoint
+  headers: {
+    'signoz-ingestion-key': '<SIGNOZ_INGESTION_KEY>',
+  },
+});
+```
+
+Replace the placeholders:
+
+- `<region>`: Your SigNoz Cloud region, for example `us`, `eu`, or `in`.
+- `<SIGNOZ_INGESTION_KEY>`: Ingestion key for your SigNoz Cloud org. See [Ingestion Keys](https://signoz.io/docs/ingestion/signoz-cloud/keys/).
+
+<Admonition type="info">
+  Using self-hosted SigNoz? Most steps are identical. To adapt this guide, update the endpoint and
+  remove the ingestion key header as shown in [Cloud →
+  Self-Hosted](https://signoz.io/docs/ingestion/cloud-vs-self-hosted/#cloud-to-self-hosted).
+</Admonition>
+
+</TabItem>
+<TabItem value="kubernetes" label="Kubernetes">
+
+If your application runs in Kubernetes with an OpenTelemetry Collector:
+
+```javascript:instrumentation.js
+const logExporter = new OTLPLogExporter({
+  url: 'http://<otel-collector-service>.<namespace>.svc.cluster.local:4318/v1/logs',
+  headers: {},
+});
+```
+
+**Find your collector service details:**
+
+```bash
+# List OTel collector services
+kubectl get svc -A | grep otel
+
+# Example output:
+# platform  signoz-otel-collector  ClusterIP  10.96.0.10  <none>  4318/TCP
+```
+
+Replace `<otel-collector-service>` and `<namespace>` with your actual values.
+
+<Admonition type="warning">
+**Avoiding Duplicate Logs**
+
+If using the `k8s-infra` chart (which auto-collects container logs), disable log collection for this application to prevent duplicates.
+
+[See how to disable log collection →](https://signoz.io/docs/opentelemetry-collection-agents/k8s/k8s-infra/install-k8s-infra/#disable-log-collection)
+
+</Admonition>
+
+See the [K8s-infra installation guide](https://signoz.io/docs/opentelemetry-collection-agents/k8s/k8s-infra/install-k8s-infra/) for more details.
+
+</TabItem>
+</Tabs>
+
+### Step 5: Run the Application
+
+Run your application:
+
+```bash
+node index.js
+```
+
+## Validate Logs
+
+Open your SigNoz instance and navigate to the Logs section to view the logs sent from your application.
+
+<figure data-zoomable align="center">
+  <img
+    src="/img/docs/logs-management/send-logs/nodejs-bunyan-logs.webp"
+    alt="Node.js Bunyan Logs"
+  />
+  <figcaption>
+    <i>Node.js Bunyan Logs</i>
+  </figcaption>
+</figure>
+
+<figure data-zoomable align="center">
+  <img
+    src="/img/docs/logs-management/send-logs/nodejs-bunyan-logs-detailed.webp"
+    alt="Node.js Bunyan Logs"
+  />
+  <figcaption>
+    <i>Node.js Bunyan Logs Detailed View</i>
+  </figcaption>
+</figure>
+
+## Troubleshooting
```

Comment:
Use ToggleHeading for troubleshooting - https://github.com/SigNoz/signoz.io/blob/main/CONTRIBUTING.md#send-data-docs-instrumentation-and-pipelines